platform/default/src/mbgl/gfx/headless_frontend.cpp

```diff
@@ -169,6 +157,24 @@ void HeadlessFrontend::renderOnce(Map&) {
     util::RunLoop::Get()->runOnce();
 }
 
+void HeadlessFrontend::renderFrame() {
```

Comment:
Should this code maybe moved to `MLNRenderFrontend`?

I know it was here before but I feel like it doesn't belong here because it is specific to `MLNRenderFrontend`.

Reply:
This code is specific to headless_frontend, which is used in render tests and now in Benchmark app. 
`MLNRenderFrontend` is not headless frontend, it is used in the MapView to render directly to the screen.

Reply:
Ah I was confusing `RendererFrontend` and `HeadlessFrontend`. Thanks.